src/pytest_cov/plugin.py

```diff
@@ -59,6 +59,9 @@ def pytest_configure(config):
     """Activate coverage plugin if appropriate."""
     if config.getvalue('cov_source'):
         if not config.pluginmanager.hasplugin('_cov'):
+            if not config.option.cov_report:
```

Comment:
Yes ;) If you comment out this line https://github.com/schlamar/pytest-cov/blob/8209d9dbfdc23f51b58a9964798f2f82d79e685c/tests/test_pytest_cov.py#L516 it has an effect ;)